Sources/EdgeNetworkHandlers/EdgeEndpoint.swift

```diff
@@ -40,9 +40,11 @@ struct EdgeEndpoint {
     ///   - requestType: the `EdgeRequestType` to be used
     ///   - environmentType: the `EdgeEnvironmentType` for the `EdgeEndpoint`
     ///   - optionalDomain: an optional custom domain for the `EdgeEndpoint`. If not set the default domain is used.
+    ///   - optionalPath: an optional path to be used to overwrite the default interact path.
```

Comment:
As the endpoint can be for interact, collect, or consent requests, can you reword to "... overwrite the default path".